apps/codeforafrica/playwright.config.js

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable import/no-extraneous-dependencies */
+module.exports = require("playwright-config-commons-ui");
```

Comment:
Need to configure webServer to run on port `3002` as well. See [`promisetracker`](https://github.com/CodeForAfrica/ui/blob/beed8171c0a43702b4b12baaf24056d1e1804df7/apps/promisetracker/playwright.config.js#L2-L14).